client/my-sites/upgrades/domain-management/list/index.jsx

```diff
@@ -383,6 +383,7 @@ export default connect( ( state, ownProps ) => {
 			{
 				cta_name: 'domain_info_notice'
 			}
-		) )
+		) ),
+		setPrimaryDomain: ( ...props ) => setPrimaryDomain( ...props )( dispatch )
```

Comment:
Super minor, the style here is reversed from: `dispatch( recordTracksEvent( ... ) )`